extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -57,77 +55,97 @@ public class ParquetTableLocation extends AbstractTableLocation {
     private static final String IMPLEMENTATION_NAME = ParquetColumnLocation.class.getSimpleName();
 
     private final ParquetInstructions readInstructions;
-    private final ParquetFileReader parquetFileReader;
-    private final int[] rowGroupIndices;
 
-    private final ParquetColumnResolver resolver;
+    private ParquetColumnResolver resolver;
```

Comment:
This needs to be guarded by initialize() too, right? Move into the section below if so.